book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_reader :rentals
+  attr_accessor :title, :author
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    @rentals.push(rental)
+  end
```

Comment:
- [ ] The add rental method should create and return a new rental, kindly implement this so the book -> rentals has the correct associations

  e.g
  ``` ruby
  def add_rental(person, date)
    Rental.new(self, person, date)
  end
  ```